benchmarks/cpp/transformer.cpp

```diff
@@ -0,0 +1,131 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <device_lower/lower2device.h>
+#include <fusion.h>
+#include <ir/all_nodes.h>
+#include <ir/builder.h>
+#include <ir/utils.h>
+#include <ops/all_ops.h>
+#include <runtime/executor.h>
+#include <scheduler/all_schedulers.h>
+#include <scheduler/utils.h>
+
+#include <benchmark/benchmark.h>
+
+#include <cuda_runtime.h>
+
+#include <sstream>
+
+#include <benchmarks/cpp/utils.h>
+#include <csrc/multidevice/utils.h>
+#include <tests/cpp/multidevice_transformer.h>
+#include <tests/cpp/utils.h>
+
+using namespace nvfuser;
+
+namespace {
```

Comment:
constexpr has internal linkage by default. There's no need to put them inside an anonymous namespace.